homeassistant/components/airzone/config_flow.py

```diff
@@ -51,13 +52,17 @@ async def async_step_user(
             )
 
             try:
-                await airzone.validate()
+                mac = await airzone.validate()
             except InvalidSystem:
                 data_schema = SYSTEM_ID_SCHEMA
                 errors[CONF_ID] = "invalid_system_id"
             except AirzoneError:
                 errors["base"] = "cannot_connect"
             else:
+                if mac:
+                    await self.async_set_unique_id(format_mac(mac))
+                    self._abort_if_unique_id_configured()
```

Comment:
```suggestion
                    self._abort_if_unique_id_configured(updates={CONF_HOST: user_input[CONF_HOST], CONF_PORT: user_input[CONF_PORT]})
```

Can we update the host/port here?

Reply:
> Can we update the host/port here?

Sure!